examples/suit_update/Makefile

```diff
@@ -0,0 +1,116 @@
+# name of your application
+APPLICATION = suit_update
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-mega2560 arduino-nano \
+                             arduino-uno b-l072z-lrwan1 chronos lsn50 msb-430 \
+                             msb-430h nucleo-f031k6 nucleo-f042k6 nucleo-l031k6 \
+                             nucleo-f030r8 nucleo-f302r8 nucleo-f303k8 \
+                             nucleo-f334r8 nucleo-l053r8 nucleo-l073rz ruuvitag \
```

Comment:
I think some boards in this list could be removed: b-l073z-lrwan1, nucleo-l073rz, lsn50, ruuvitag and thingy52. I'm able to build the slots for STM32 l07x based boards and the nordic boards uses the same CPU as nrf52dk.

Reply:
I had added them as CI failed. Just tried b-l072z-lrwan1, it fails with ram overflow.

Reply:
but ruuvitag and thingy52 can be removed (nrf52832 based, like nrf52dk)

Reply:
ah, thingy and ruuvitag fail because ethos conflicts with stdio_rtt.